.github/workflows/release-build.yml

```diff
@@ -55,22 +60,3 @@ jobs:
           asset_path: otelagent/build/libs/aws-opentelemetry-agent-${{ github.event.inputs.version }}.jar
           asset_name: aws-opentelemetry-agent.jar
           asset_content_type: application/java-archive
-
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: us-west-2
-
-      - name: Login to Amazon ECR
-        id: login-ecr
-        uses: aws-actions/amazon-ecr-login@v1
-
-      - name: Push image to Amazon ECR
```

Comment:
Using the `jib` command does this push instead now right?

Reply:
Yeah